README.md

```diff
@@ -2,35 +2,67 @@
 <img src="images/democracy-earth.png" width="400" title="Democracy Earth Foundation">
 </p>
 
-# Liquid Democracy
+# Liquid Democracy as a smart contract
 
 > 🚨🚧 _Everything in this repo is still a work in progress and higly experimental. It is not secure to use any of this code in production (mainnet) until proper security audits have been conducted._
 
-This is a smart contract implementation of a basic liquid democracy with three initial main functionalities: `vote()`, `delegate()` and `revoke()`. It mainly consists of `LiquidDemocracy.sol` and its corresponding test file `LiquidDemocracy.test.js`. This is the starting point of a roadmap that seeks to bridge Sovereign app with blockchain features. Please make sure you check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) and the [milestones](https://github.com/DemocracyEarth/contracts/milestones) for more information.
+This is a smart contract implementation (Dapp) in the Ethereum blockchain of a [liquid democracy](https://github.com/DemocracyEarth/paper) with three initial main functionalities: `vote()`, `delegate()` and `revoke()`. It mainly consists of `LiquidDemocracy.sol` and its corresponding test file `LiquidDemocracy.test.js`. This is the starting point of a roadmap that seeks to bridge [Sovereign app](https://github.com/DemocracyEarth/sovereign) with blockchain features. Please make sure you check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) and the [milestones](https://github.com/DemocracyEarth/dapp/milestones) for more information.
 
-## How to collaborate
+The most recent version of the dapp is uploaded in [http://dapp.democracy.earth](). 
+For desktop instal metamask or a metamask-supported browser to be able to interact with it.
+For mobile, use a dapp browser / wallet like [Toshi](https://www.toshi.org/)
 
-PRs and feedback are welcomed. A good place to start is to review the current [open issues](https://github.com/DemocracyEarth/contracts/issues), please feel free to engage with us by commenting in a specific issue. Remember you can also check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) to understand what's being worked on now and what is coming up next.
+## Contributing
 
-## How to test locally
+PRs and feedback are welcomed. A good place to start is to review the current [open issues](https://github.com/DemocracyEarth/dapp/issues), please feel free to engage with us by commenting in a specific issue. Remember you can also check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) to understand what's being worked on now and what is coming up next.
```

Comment:
Should we keep the Waffle board link there? I think we should but just asking as there's a [Github project board](https://github.com/DemocracyEarth/dapp/projects/3#card-9903310) too. Waffle board is more encompassing, think it makes sense as a guide

Reply:
@medied I don't think we are using the waffle board anymore, I think we should just point people to the latest issues.

Reply:
let's point to github project board then

Reply:
Cool yea it doesn't really matter which one it is, as long as it's the active one we are actually using

Reply:
Updated with 8b9b68a8fe373c499c6487c00a43e0b689a2791d